### ISSUE TITLE
Valid accessibilityFeature values generate Ace moderate violation

### ISSUE THREAD

**Issue by FeniXX-prod** [daisy/ace#242]
Version of Ace impacted by the issue : 1.1.
The list of values for the accessibilityFeature property recommended by the Daisy Consortium is more restrictive than the W3C's.
e.g. the use of the following values will generate a "moderate violation" in Ace ("'schema: accessibilityFeature' metadata must be set to one of the expected values") : printPageNumbers, readingOrder, tableOfContents, unlocked.

**Comment by rdeltour** [comment]
We should support the whole W3C list, see the list in [the source code](https://github.com/daisy/ace/blob/v1.1.1/packages/ace-core/src/checker/checker-epub.js#L56:L86).

I think the issue is that you declare all these features as a comma-separated list, when you should instead repeat the property for each feature?

**Comment by FeniXX-prod** [comment]
I should have wait a couple minutes more before writing the issue. Fixed in 1.1.1.
(the property was repeated for each feature : 
<meta property="schema:accessibilityFeature">printPageNumbers</meta>
<meta property="schema:accessibilityFeature">readingOrder</meta>
<meta property="schema:accessibilityFeature">tableOfContents</meta>
<meta property="schema:accessibilityFeature">unlocked</meta>)
Thanks.

**Comment by mpakarlsson** [comment]
We are in the process of adding accessibility meta data into our Save as EPUB3 tool and bounced into a very similar issue with moderate violation on 'accessibilityFeature'. I'm using Ace, by Daisy 1.0.0-rc.1 and according to the report, it's uses Ace 1.1.1.

Example EPUB3 file:
https://www.dropbox.com/s/xfmi7n3o0v81nml/Alice.epub?dl=0

Not sure if the error is in the EPUB file, or in the validation tool - please advice.

**Comment by rdeltour** [comment]
@mpakarlsson it seems the issue comes from the value "resizeText", which is used in your EPUB but not listed as an acceptable value on the authoritative [W3C WebSchemas/Accessibility wiki page](https://www.w3.org/wiki/WebSchemas/Accessibility#Accessibility_terms_.28Version_2.0.29).

According to that page, the way to use this metadata is to associate it to the "displayTransformability" value (e.g. "displayTransformability/font-size", which by the way might not pass Ace currently, to be investigated). 
"displayTransformability/resizeText" is used in one of the example, but this doesn't seem to be conforming to the spec (bold highlight is mine):

> This property can be modified to identify the specific display properties that allow meaningful control. **Modifiers should take the form of CSS property names**, even if CSS is not the document styling format

**Comment by mpakarlsson** [comment]
@rdeltour Thanks for the quick response, much appreciated!
